bootstrap/cell/main.tf

```
@@ -58,4 +58,8 @@ module "kupo_instances" {
       value    = "consistent"
     }
   ])
+  node_affinity = each.value.node_affinity != null ? each.value.node_affinity : {
+    required_during_scheduling_ignored_during_execution  = {}
+    preferred_during_scheduling_ignored_during_execution = []
+  }
```

NIT: I think using [`coalesce`](https://developer.hashicorp.com/terraform/language/functions/coalesce) here might be clearer.

```suggestion
  node_affinity = coalesce(each.value.node_affinity, {
    required_during_scheduling_ignored_during_execution  = {}
    preferred_during_scheduling_ignored_during_execution = []
  })
```